internal/spokes/spokes_test.go

```diff
@@ -28,7 +28,7 @@ func TestCheckHiddenRefs(t *testing.T) {
 		{"refs/remotes/origin/main", hiddenRefs, false},
 		{"refs/__gh__/pull/99986/rebase", hiddenRefs, true},
 		{"refs/gh/merge_queue/156066/6e33e3a2c52017bec941ffd6f15c20a1ae002ad9", hiddenRefs, true},
-		{"refs/pull/95628/head", hiddenRefs, true},
+		{"refs/__github__/svn/branch-1", hiddenRefs, false},
```

Comment:
```suggestion
		{"refs/__gh__/svn/branch-1", hiddenRefs, false},
```

To match if you take my suggestion above.